app/assets/stylesheets/layout.scss

```diff
@@ -62,6 +62,14 @@ body{
     background: $main-panel-background;
     margin: 1em;
 
+    a:hover{
+      background-color: inherit;
+    }
+
+    .field_with_errors{
```

Comment:
Rails-native, it's just `red;` e.g. `ff0000`, which isn't a good contrast ratio with black text (5.3:1).